script/update-typedoc-link.js

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const fs = require('fs-extra');
+
+const walk = function(dir) {
+  let results = [];
+  let list = fs.readdirSync(dir);
+  list.forEach(function(file) {
+    file = dir + '/' + file;
+    let stat = fs.statSync(file);
+    if (stat && stat.isDirectory()) {
+      results = results.concat(walk(file));
+    } else {
+      results.push(file);
+    }
+  });
+  return results;
+};
+
+walk('docs')
+  .filter(fn => fn.endsWith('.html'))
+  .forEach(file => {
+    fs.readFile(file, 'utf8', (err, data) => {
+      if (err) return console.error(err);
+
+      let result = data.replace(/blob\/[0-9a-f]{5,40}\/src/g, 'blob/master/src');
```

Comment:
Let's remove `src` from the regex if possible so that this rewrite will apply to any URL if in the future there are URLs to files outside of `src`.

Reply:
Fixed.